IETF-RFC.md

```diff
@@ -495,7 +516,9 @@ Step 4: If not, try a HTTP GET with `https://<fqdn>/ocm-provider` as
 the URL instead.
 Step 5: If that results in a valid HTTP response with a valid JSON
 response body within reasonable time, go to step 7.
-Step 6: If not, fail.
+Step 6: If not, fail. Implementations MAY fallback to `http` instead
+`https` in testing setups and retry steps 2-5 using `http`. This SHOULD
+only be allowed as Opt-In.
```

Comment:
```suggestion
Step 6: If not, fail. Implementations MAY fallback to HTTP instead
of HTTPS in testing setups and retry steps 2-5, in particular when
an optional port is given in the address.
```
I don't think we buy much in stating the Opt-in, rather the issue is this is for implementations to do in the context of testing, using custom ports as they wish.